debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/SourceInfo.java

```diff
@@ -138,6 +146,10 @@ public Long xmin() {
         return this.xmin;
     }
 
+    public Lsn[] sequence() {
```

Comment:
This should be pulled up to the parent class. The signature should return `String[]` and the default implementation null. This class will just override the default.

Reply:
Hi @jpechane -- I kept `sequence()` on `SourceInfo` because it's needed by `AbstractSourceInfoStructMaker`--I was uncertain how I'd grab `EventMetadataProvider`. If this is incorrect, I'd be happy to fix it!

Reply:
You are right, thre is no direct link. That would probably be nice to do it as a separate PR but now it is fine to keep it as is.